dev/script/es-ssh-ssm-tunnel.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+set -e
+
+function HELP {
+>&2 cat << EOF
+
+  Usage: ${0} -t ES-HOST [-s TEST] [-u ubuntu]
+
+  This script sets up an ssh tunnel from localhost port 9300 to the ElasticSearch provided on port 9300
+
+    -v            Verbose ssh output
+
+    -h            Displays this help message. No further functions are
+                  performed.
+
+EOF
+exit 1
+}
+
+BACKGROUND="-f"
+VERBOSE=""
+
+# Process options
+while getopts u:s:t:hv FLAG; do
+  case $FLAG in
+    h)  #show help
+      HELP
+      ;;
+    v)
+      VERBOSE="-v"
+      ;;
+    s)
+      STAGE=$OPTARG
+  esac
+done
+shift $((OPTIND-1))
+
+
+if [ -z "${STAGE}" ]; then
+  STAGE="TEST"
+fi
+echo "📡 getting ELB name from AWS"
+
+ELB_NAME=""
```

Comment:
should this be the empty string?

Reply:
This step isn't needed as I got the direct tunnel working, will remove.